packages/core-ethereum/src/types/ticket.spec.ts

```diff
@@ -68,7 +71,7 @@ describe('test ticket construction', function () {
     const epoch = UINT256.fromString('2')
     const index = UINT256.fromString('1')
     const amount = new Balance(new BN('000000000de0b6b3a7640000', 16))
-    const winProb = new Hash(stringToU8a('0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff'))
+    const winProb = Ticket.floatToUint256(1)
```

Comment:
perhaps should be UInt256.fromNumber() instead?

Reply:
I don't neither `floatToUint256` or `fromNumber` TBH, what would word would best describe what we are doing here? we are "upscaling" a float to UINT256's max number

Reply:
it's a js number, not a `float`, and we are creating a Uint256 from it.

On the other hand, I misread this - if we are creating from a 0-1 range, then it should be:

Uint256.fromProbability(x) and throw if x > 1